maggy/optimizer/abstractoptimizer.py

```diff
@@ -14,15 +14,14 @@
 #   limitations under the License.
 #
 
+import numpy as np
```

Comment:
standard library imports should always go first. make sure to use flake8 for linting

Reply:
I used flake8 for linting. Have I done something wrong?

Reply:
No not necessarily. In most files you touched, the imports are reordered, alphabetically. Either you have manually changed the order or I was guessing that your IDE does it because it uses another linter than flake8. That's why you should make sure your IDE uses also flake8 to be consistent.

If you changed the order manually, then we should stick to PEP8 https://www.python.org/dev/peps/pep-0008/#imports see the second bullet point.

in this case it would mean:
```
import time
from abc import ABC, abstractmethod
from datetime import datetime

import numpy as np

from maggy.core.environment_singleton import environment_singleton
from maggy.pruner import Hyperband
from maggy.trial import Trial
```
Numpy is not part of the standard library.